mm2src/coins/eth/eth_tests.rs

```diff
@@ -126,6 +126,7 @@ fn eth_coin_from_keypair(
     let ticker = match coin_type {
         EthCoinType::Eth => "ETH".to_string(),
         EthCoinType::Erc20 { .. } => "JST".to_string(),
+        EthCoinType::Nft { .. } => unreachable!(),
```

Comment:
How do we ensure that?

Reply:
> May I know how do we ensure that?

Thanks for the catch, I think the `unreachable` was left from previous version of `EthCoinType::Nft`, where `Nft` wasnt structure with `platform` field.
Will use `platform`coin name from `Nft { platform }` for ticker.